### PR TITLE
added instructions for postgres and how to fix issues when creating superuser

### PR DESCRIPTION
**Pull request by ignujee**
The instructions in installing and configuring postgres, setting the necessary environment and running python manage createsuperuser has been detailed

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -85,6 +131,34 @@ Setting Up Your Users
...
+
+run this command again
+
+`python manage.py createsuperuser`
+
+This will now prompt for the following details - Ignore any warnings.
+
+- username: give the username here
+- usertype:Give the value `10` [5 for doctor, 10 for hospital staff/hospital administrator, 15 for patient, 20 for volunteer]
+- gender: 1 for male, 2 for female, 3 for other
```
Comment (abhiabhi94) [inline README.rst]: non-binary is the term not "others"

File: README.rst [hunk 2]
```diff
@@ -27,6 +27,52 @@ Set up Local environment
...
+
+`sudo -u postgres psql`
+
+In the postgres# shell type `\\password postgres` to change the password
+
+Exit psql
+
+`\\q`
+
+Edit `/etc/postgresql/9.6/main/pg_hba.conf` and change:
```
Comment (farisshajahan) [inline README.rst]: Change 9.6 to <postgres version>....
The version being used may be different

**Comment by farisshajahan** [comment]
Also format the code lines.

File: README.rst [hunk 3]
```diff
@@ -85,6 +131,34 @@ Setting Up Your Users
...
+run this command again
+
+`python manage.py createsuperuser`
+
+This will now prompt for the following details - Ignore any warnings.
+
+- username: give the username here
+- usertype:Give the value `10` [5 for doctor, 10 for hospital staff/hospital administrator, 15 for patient, 20 for volunteer]
+- gender: 1 for male, 2 for female, 3 for other
+- email: give e-mailid
```
Comment (mgaitan) [inline README.rst]:
```suggestion
- email: give e-mail id
```

File: README.rst (hunk 1 above)
Comment (ignujee) [reply README.rst]: where should I use non-binary instead of others? In the gender: expansion?

**Pull request closed without merging**